app.py

```diff
@@ -168,12 +172,14 @@ def randomizePage(data, sessionDict):
         randomizer.setRewards(levelChoice = sessionDict["levelChoice"], betterJunk=("Better Junk" in sessionDict["seedModifiers"]))
         randomizer.setLevels(sessionDict["soraExpMult"], formExpMult = sessionDict["formExpMult"], statsList = SeedModifier.glassCannon("Glass Cannon" in sessionDict["seedModifiers"]))
         randomizer.setBonusStats()
-        # if not randomizer.validateSeed(startingInventory = sessionDict["startingInventory"]):
-        #     print("ERROR: Seed is not completable! Trying another seed...")
-        #     characters = string.ascii_letters + string.digits
-        #     sessionDict['seed'] = (''.join(random.choice(characters) for i in range(30)))
-        #     randomizePage(data,sessionDict)
-        #     return
+        if not randomizer.validateSeed(startingInventory = sessionDict["startingInventory"], reverseRando=("Reverse Rando" in sessionDict["seedModifiers"])):
+            print("ERROR: Seed is not completable! Trying another seed...")
```

Comment:
I can write this but I would like to have a integration test that generates X seeds with maxItemLogic on to verify  that the invalid_checks rules are not violated.

I think that might require an adjustment to the seed method (so it can be called in test mode or normal mode)